includes/class-wc-amazon-payments-advanced-rest-api-controller.php

```diff
@@ -631,14 +625,14 @@ protected function get_missing_capture_id_request_error( $order_post ) {
 	 * @return WP_Post|WP_Error Post object if it's valid, WP_Error if it's invalid.
 	 */
 	protected function is_valid_order( $order_id ) {
-		$order_post = get_post( (int) $order_id );
+		$or = wc_get_order( (int) $order_id );
```

Comment:
Can we give this variable a better name, please?

Reply:
done!